packages/@okta/vuepress-site/docs/reference/api/threat-insight/index.md

```diff
@@ -0,0 +1,97 @@
+---
+title: ThreatInsight
+category: management
+---
+
+# ThreatInsight Configuration API
+
+[Okta ThreatInsight](https://help.okta.com/en/prod/Content/Topics/Security/security-threat-insight.htm) maintains a constantly evolving list of IPs that exhibit suspicious behaviors suggestive of malicious activity. Authentication requests associated with an IP in this list can be logged in [System Log](https://help.okta.com/en/prod/Content/Topics/Reports/Reports_SysLog.htm?cshid=ext_Reports_SysLog) and blocked. The Okta ThreatInsight Configuration API provides operations to manage your ThreatInsight configuration.
+
+
+## ThreatInsight Configuration object
+
+| Field Name     | Description                                                         	| Data Type                                     | Required      | Max Length    |
+| :------------- | :------------------------------------------------------------------	| :-------------------------------------------- | :------------ | :------------ |
+| action         | Specifies how Okta would respond to authentication requests from suspicious IPs. Value is one of `none`, `audit`, `block`. With `none`, ThreatInsight will be disabled; with `audit`, Okta will log the suspicious requests in System Log; with `block`, Okta will log the suspicious requests in System Log and block the requests. | String                                        | Yes		| N/A           |
+| excludeZones   | Accepts a list of [Network Zone](/docs/reference/api/zones/) IDs. IPs in the excluded Network Zones will not be logged or blocked by Okta ThreatInsight and will proceed to Sign On rules evaluation. This will ensure that traffic from known, trusted IPs is not accidentally logged or blocked.	| List	                                        | No		| N/A           |
+
+
+## ThreatInsight Configuration API Operations
+
+### Update ThreatInsight configuration
+
+<ApiOperation method="post" url="/api/v1/threats/configuration" />
+
+Update ThreatInsight configuration
+
+#### Request Example
```

Comment:
Request example

Reply:
hmm I see we are capitalizing example across dev doc, should we keep it for consistency purpose?

Reply:
We are in the process of migrating everything to sentence case style for headings. The style information is here:
https://github.com/okta/okta-developer-docs/wiki/Style-Guide#headings

There are a number of API Reference pages that have already been edited for style and updated, such as /apps
https://developer.okta.com/docs/reference/api/apps/

Also, there is a template for creating API References found here that follow the style guidelines as well:
https://oktawiki.atlassian.net/wiki/spaces/DOC/pages/748062101/REST+API+Reference+Markdown+Template

Reply:
Makes sense, will update, thanks!